sources/packages/backend/apps/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -155,14 +151,12 @@ export class InstitutionLocationInstitutionsController extends BaseController {
   @Get(":locationId/active-applications")
   async getActiveApplications(
     @Param("locationId", ParseIntPipe) locationId: number,
-    @Query() pagination: ApplicationStatusPaginationOptionsAPIInDTO,
-    @Query("archived", new DefaultValuePipe(false), ParseBoolPipe)
-    archived: boolean,
+    @Query() queryStringDTO: ActiveApplicationSummaryAPIQueryStringDTO,
   ): Promise<PaginatedResultsAPIOutDTO<ActiveApplicationSummaryAPIOutDTO>> {
     const applications = await this.applicationService.getActiveApplications(
       locationId,
-      pagination,
-      archived,
+      queryStringDTO,
```

Comment:
Please update the comment with modified param.

Also param name queryStringDTO seems to be very generic.

Reply:
I will update the comments,
The `queryStringDTO` was meant to be generic. It is supposed to represent any query string received by the endpoint, in this case, paginations options + archive flag.
I avoided using a name that represents "paginations options + archive flag" thinking that in the future if a new parameter is added we should not rename the parameter again.